src/main/java/com/github/tomakehurst/wiremock/global/GlobalSettings.java

```diff
@@ -26,21 +26,21 @@ public class GlobalSettings {
   private final DelayDistribution delayDistribution;
   private final Parameters extended;
 
+  private final Boolean proxyPassThrough;
```

Comment:
I think this should be the primitive `boolean` here and elsewhere it's used as I don't think there's any reason for it to be null, and it's consistent with other flags in the configuration.

Reply:
@tomakehurst incorporated this change